src/engage/enrollments/enrollment.action.ts

```diff
@@ -0,0 +1,90 @@
+import {ZapierItem} from '../../types'
+import {createEnrollment, stopEnrollment} from '../common'
+import {createActionDetails} from '../../utils/operations'
+import {userSearches, userTriggers} from '../../users/keys'
+import {enrollmentActions} from '../keys'
+import EnrollmentResource from './enrollment.resource'
+
+export const CreateEnrollmentAction: ZapierItem = {
+    key: enrollmentActions.createEnrollmentAction,
+    noun: 'Enrollment',
+    display: {
+        label: 'Create Enrollment',
+        description: 'Creates a new enrollment.',
+        important: true
+    },
+    operation: {
+        resource: EnrollmentResource.key,
+        inputFields: [
+            {
+                key: 'sequence_id',
+                label: 'Sequence',
+                type: 'integer',
+                required: true
+            },
+            {
+                key: 'sell_resource',
+                label: 'Sell Resource',
+                children: [
+                    {
+                        key: 'resource_id',
```

Comment:
Don't you want to make this field a dropdown? This will give users an ability to select lead from the dropdown
examples can be found for tasks and notes here
https://github.com/zendesk/zendesk-sell-for-zapier/blob/master/src/notesTasks/tasks/task.action.ts#L48
https://github.com/zendesk/zendesk-sell-for-zapier/blob/master/src/notesTasks/common.ts#L109

Reply:
I've added simple dropdown search on leads (also replaced ID fields with dropdowns for Enrollments and Sequences where applicable)